Gemfile

```diff
@@ -1,18 +1,18 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.6'
+gem 'rails', '~> 5.0.1'
 gem 'pg', '~> 0.18.4'
 gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'active_model_serializers'
 gem 'responders'
 gem 'rack-haproxy_status', '~> 0.8.1'
 
-gem 'sprockets-rails', '~> 3.0.4'
+gem 'sprockets-rails'
```

Comment:
Er vanuit gaande dat ze semantic versionen zou je nooit de build vast moeten zetten. dus '~> 3.0', niet '~> 3.0.4', gezien alles binnen een mayor version compatible moet zijn.

Ben altijd wel fan geweest van mayor version vast zetten. Zodat je idd elke dag bundle update zou moeten kunnen draaien. Doch is er een punt te maken dat als je test-suite goed genoeg is en gemnasion verschillende configuraties uitprobeerd en test, dat het onderhand niet meer nodig is.